site/pages/type-of-consequence/__snapshots__/type-of-consequence.test.tsx.snap

```diff
@@ -1,5 +1,353 @@
+                    className="govuk-fieldset__legend govuk-!-padding-top-3"
+                  >
+                    <span
+                      className="govuk-heading-s govuk-!-margin-bottom-0"
+                    >
+                      Select consequence type
+                    </span>
+                  </legend>
+                  <div
+                    className=""
+                  >
+                    <div
+                      className="govuk-radios"
+                      data-module="govuk-radios"
+                    >
+                      <div
+                        className="govuk-radios__item govuk-!-margin-bottom-1"
+                      >
+                        <input
+                          className="govuk-radios__input"
+                          data-aria-controls="consequence-type-services-conditional"
+                          defaultChecked={false}
+                          id="consequence-type-services"
+                          name="consequenceType"
+                          onChange={[Function]}
+                          type="radio"
+                          value="services"
+                        />
+                        <label
+                          className="govuk-label govuk-radios__label"
+                          htmlFor="consequence-type-services"
+                        >
+                          Services
+                        </label>
+                      </div>
+                      <div
+                        className="govuk-radios__item"
+                      >
+                        <input
+                          className="govuk-radios__input"
+                          data-aria-controls="consequence-type-operatorWide-conditional"
+                          defaultChecked={false}
+                          id="consequence-type-operatorWide"
+                          name="consequenceType"
+                          onChange={[Function]}
+                          type="radio"
+                          value="operatorWide"
+                        />
+                        <label
+                          className="govuk-label govuk-radios__label"
+                          htmlFor="consequence-type-operatorWide"
+                        >
+                          Operator wide
+                        </label>
+                      </div>
+                    </div>
+                  </div>
+                </fieldset>
+              </div>
+              <input
+                name="disruptionId"
+                type="hidden"
+              />
+              <input
+                name="consequenceIndex"
+                type="hidden"
+              />
+              <div
+                className="govuk-button-group"
+              >
+                <button
+                  className="govuk-button"
+                  data-module="govuk-button"
+                >
+                  Save and continue
```

Comment:
When i press save and continue after entering all fields it does not navigate to the review page